### ISSUE TITLE
Implementation and deployment of Ghana's national curriculum at Toolforge

### ISSUE THREAD

**Issue by GonzaFP** [wikicurricula-uy/wikicurricula-boilerplate#10]
I am writing documentation on how to deploy Wikicurricula boilerplate in Toolforge and also deploying Ghana's national curriculum in Toolfrage

**Comment by GonzaFP** [comment]
Hello @EneHacheCe , @SannitaSSJ , @fernandouval , @nnatahihechi  and @wikicurricula-uy . I have implemented and deployed Wikicurricula for [Ghana's national curriculum](https://w.wiki/7Zge), with reference to the English Wikipedia. Furthermore, I have documented the process in the README.md.
The most cumbersome task was to Understand Toolforge - its purpose, how to create different accounts associated with it and how to deploy a tool to Toolforge.

Right now, the tool's user interface is basic, I suggest we add more functionality hence I am going to open new issues.
I will appreciate your feedback!
